template/frameworks/ava/.babelrc

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Can we use [`babel.config.js`](https://babeljs.io/docs/en/configuration#babelconfigjs) for this as a root config file ?